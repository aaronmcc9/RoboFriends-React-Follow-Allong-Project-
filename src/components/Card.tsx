import react from 'react';
import { Robot } from '../containers/App';

interface CardProps{
    robot: Robot
}

const Card = ({robot}: CardProps) => {
    return (
        <div className='bg-light-green dib br3 ma3 grow bw2 shadow-5'>
            <img alt='robots' src={`https://robohash.org/${robot.id}?size=200x200`}/>
            <div>
                <h2>{robot.name}</h2>
                <p>{robot.email}</p>
            </div>
        </div>
    );
}

export default Card;