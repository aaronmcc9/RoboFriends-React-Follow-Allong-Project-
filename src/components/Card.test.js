import { shallow, mount, render } from 'enzyme';
import React from 'react';
import Card from './Card';

it('expects to render card component', () => {
    expect(shallow(<Card />)).toMatchSnapshot();
})