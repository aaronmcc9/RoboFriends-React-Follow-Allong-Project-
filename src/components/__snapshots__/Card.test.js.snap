// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render card component 1`] = `ShallowWrapper {}`;
